quiche/src/lib.rs

```diff
@@ -8042,6 +8180,8 @@ pub struct TransportParams {
     pub retry_source_connection_id: Option<ConnectionId<'static>>,
     /// DATAGRAM frame extension parameter, if any.
     pub max_datagram_frame_size: Option<u64>,
+    /// Unknown peer transport parameters and values, if any.
+    pub unknown_params: UnknownTransportParameters,
```

Comment:
Why not just make this an option? It would also simplify the decode implementation a bit, so instead of:
```
if track_unknown_parameters {
```
you could just do
```
if tp.unknown_params.is_some()
```